ext/odbc/php_odbc.c

```diff
@@ -2680,36 +2712,42 @@ PHP_FUNCTION(odbc_errormsg)
 */
 PHP_FUNCTION(odbc_setoption)
 {
-	odbc_connection *conn;
+	odbc_link *link;
 	odbc_result	*result;
 	RETCODE rc;
 	zval *pv_handle;
 	zend_long pv_which, pv_opt, pv_val;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "rlll", &pv_handle, &pv_which, &pv_opt, &pv_val) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "olll", &pv_handle, &pv_which, &pv_opt, &pv_val) == FAILURE) {
```

Comment:
Maybe?
```suggestion
	if (zend_parse_parameters(ZEND_NUM_ARGS(), "Olll", &pv_handle, odbc_connection_ce, &pv_which, &pv_opt, &pv_val) == FAILURE) {
```

Reply:
It's not possible unfortunately, since the function also accepts a `result_ce` when the 3rd argument has a value of `2`.